src/net/http/example_test.go

```diff
@@ -191,3 +192,19 @@ func ExampleNotFoundHandler() {
 
 	log.Fatal(http.ListenAndServe(":8080", mux))
 }
+
+func ExamplePost() {
+	r, e := http.Post("http://127.0.0.1/count", "text/plain", strings.NewReader("POST data"))
```

Comment:
I prefer `err` than `e`, I know, it's an example test but `err` would be better for an example.

Reply:
I agree with @AnikHasibul . It's go convention to use `err` instead of `e`.

Reply:
You should also give all of the variables more meaningful names.

```suggestion
func ExamplePost() {
	req, err := http.Post("http://127.0.0.1/count", "text/plain", strings.NewReader("POST data"))
        defer req.Body.Close()
	if err != nil {
		fmt.Println(err.Error())
	}
 	body, err := ioutil.ReadAll(req.Body)
	if err != nil {
		fmt.Println(err.Error())
	}
 	fmt.Printf("%s\n", body)
 }	
```